algosdk/atomic_transaction_composer.py

```diff
@@ -7,10 +7,10 @@
     List,
     Dict,
     Optional,
+    Sequence,
```

Comment:
@algochoi It's a good callout about `typing.Sequence` deprecation.  In https://github.com/jdtzmn/py-algorand-sdk/pull/3/commits/8644dffd073ba7b5925be0d8918aea588022a77a, I found a way to retain `List` usage.

I realize 3.9 deprecates https://docs.python.org/3/library/typing.html#typing.List.  I see a workaround in https://peps.python.org/pep-0585/#implementation, but it implies changing a large portion of the repo.  Unless you suggest otherwise, my preference is to limit this PR's scope.